tests/unit/test_xblock.py

```diff
@@ -116,7 +77,7 @@ def test_lti_passport(self, lti_id, expected_result):
         ('', ''),
         ('missing', ''),
         ('dalite-ng-1', "http://first.url:8080/lti/"),
-        ('dalite-ng-2', "https://other.url:8080/lti/")
+        ('dalite-ng-2', "http://other.url/lti/")
```

Comment:
Added this case and a bunch of other here: https://github.com/open-craft/xblock-dalite/pull/2/commits/2f0039a07a0efca9baf21dafa4af06b9cd554e76#diff-6ac57e7e8afa8bd1d6836a7f77908adeR20